xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -874,9 +1310,19 @@
         <Parameter Name="disableConformanceModeChecks" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="disableConformanceModeChecks">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="disableConformanceModeChecks">Disable conformance mode validation for the skipped value, equivalent to using <see cref="F:System.Formats.Cbor.CborConformanceMode.Lax" />.</param>
```

Comment:
ditto

Reply:
```suggestion
        <param name="disableConformanceModeChecks"><see langword="true" /> to disable conformance mode validation for the skipped values, equivalent to using <see cref="F:System.Formats.Cbor.CborConformanceMode.Lax" />; otherwise, <see langword="false" />.</param>
```